### ISSUE TITLE
Fix broken links to chapters within NCEA curriculum guides

### ISSUE THREAD

**Issue by JackMorganNZ** [uccser/cs-field-guide#354]
## Description

There are many broken links to chapters within the NCEA curriculum guides, for example:

```
[2.1 - What’s the bigger picture?](algorithms.html#whats-the-big-picture)
```

which should be corrected to:

```
[2.1 - What’s the bigger picture?](chapters/algorithms.html#whats-the-big-picture)
```
### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the issue._
- [x] I have linked any relevant [existing issues/suggestions](https://github.com/uccser/cs-field-guide/issues) in the description above (include `#???` in your description to reference an issue, where `???` is the issue number)
- [x] I have applied relevant labels

**Comment by microlith57** [comment]
This is still an issue. In [the NCEA curiculum guide page](http://csfieldguide.org.nz/en/curriculum-guides/ncea/index.html) some links are broken - `Algorithms - Searching (Merit/Excellence)` for example leads to a 404 page.

**Comment by JackMorganNZ** [comment]
Thanks @microlith57, I fixed the broken links but then broke the working ones by merging in some old redesign work. I'll get this fixed in the next 12 hours (hopefully fairly quickly).

**Comment by JackMorganNZ** [comment]
@microlith57 Creating a new issue (#483) for the bug you found as it's a slightly different error to this one.

**Comment by JackMorganNZ** [comment]
I've closed the new issue #438 as I've deployed version `2.10.1`. Should all be fixed now!
